optaplanner-examples/src/main/java/org/optaplanner/examples/conferencescheduling/persistence/ConferenceSchedulingRESTEndpoints.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.examples.conferencescheduling.persistence;
+
+public class ConferenceSchedulingRESTEndpoints {
```

Comment:
Given the fact you already hard-code the URLs in `ConferenceSchedulingImporter`, how about making this an Enum?

Reply:
The goal is to eventually get these values from the user, but I haven't implemented the GUI interface for this yet.

Reply:
Ok

Reply:
baseUrl needs to come from the user, for example it could be the REST url of devoxx belgium, devoxx france, voxxed zurich or any of the other conferences that use the devoxx cfp technology.

roomsEndpoint etc aren't variable: this entire code is against the "cfp-devoxx" rest api, for example the structure of the expected JSON files and therefore YAGTNI for the paths to be variable. Given a url "foo.com/rest/api/bar/boo/rooms" the path is "bar/boo/rooms", hard code that "bar/boo/rooms", directly as a string where you need it. If you need it at 2 placed, make it a constant, otherwise don't.

Rename the classes to signal that this an importer specifically for the rest api of cfp-devoxx, and add a link to https://github.com/nicmarti/cfp-devoxx in the javadoc of that class.